src/clj/game/cards/basic.clj

```diff
@@ -125,7 +125,9 @@
                 :effect (req (trigger-event state side :runner-click-draw (-> @state side :deck (nth 0)))
                              (swap! state update-in [:stats side :click :draw] (fnil inc 0))
                              (play-sfx state side "click-card")
-                             (draw state side eid 1))}
+                             (let [bonus-click-draws (get-in @state [:bonus :click-draw] 0)]
+                               (swap! state update :bonus dissoc :click-draw);; clear bonus draws
```

Comment:
If this needs to be cleared, write a function for it please. That way we only have one place to change if the key changes.